src/tooltip.js

```diff
@@ -39,17 +39,39 @@ nv.models.tooltip = function() {
     ;
 
     /*
+     * If a parent of the container has a translate transformation, fix the
+     * positioning. The fixed position of the tooltip will be based starting
+     * at this element.
+     */
+    var fixTranslation = function(pos) {
+        var obj = chartContainer;
+
+        while(obj && obj.style) {
+            if(getComputedStyle(obj).transform != 'none') {
+                var rect = obj.getBoundingClientRect();
+                pos.left -= rect.left;
+                pos.top -= rect.top;
+                break;
+            }
+
+            obj = obj.parentNode;
```

Comment:
This is always going to walk up the whole object tree every time it adjusts the tooltip position... which is a lot...  What was getBoundingClientRect().top and left doing that was wrong exactly?  From what I can tell it is supposed to take translate into account, there's even a bugzilla bug about it that they fixed at one point.